test/startsWith.test.js

```diff
@@ -0,0 +1,18 @@
+import test from 'ava'
+import {startsWith} from '../src'
+
+test('check correct input', t => {
+  const string = 'First PR'
+  const head = 'Fir'
+  const actual = startsWith(string, head)
+  const expected = true;
+  t.deepEqual(actual, expected)
```

Comment:
Could you change this to `t.true(actual)` and maybe rename the variable to `result`